app/code/Magento/Inventory/Model/ReservationRepository.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Inventory\Model;
+
+use Magento\Framework\Api\SearchCriteriaInterface;
+use Magento\Inventory\Model\Reservation\Command\GetListInterface;
+use Magento\InventoryApi\Api\ReservationRepositoryInterface;
+
+/**
+ * @inheritdoc
+ */
+class ReservationRepository implements ReservationRepositoryInterface
```

Comment:
Reading the [API and SPI](https://github.com/magento-engcom/magento2/wiki/Service-Contracts#api-and-spi-segregation) paragraph of Service Contracts page, it seems to me that Repository and GetList are different objects; Repository is an API implementing facade pattern and proxying to GetList, that is an SPI class command.
Let me know.